package.json

```diff
@@ -68,13 +68,10 @@
     ]
   },
   "pre-commit": "lint:staged",
-  "resolutions": {
```

Comment:
This feature requires an additional package called https://github.com/rogeriochaves/npm-force-resolutions

Reply:
Removed the property since this was not working.